appservice/src/deploy/localGitDeploy.ts

```diff
@@ -45,14 +48,14 @@ export async function localGitDeploy(client: SiteClient, fsPath: string): Promis
             }
             return undefined;
             // tslint:disable-next-line:no-unsafe-any
-        } else if (err.message.indexOf('error: failed to push') >= 0) {
+        } else if (parsedError.message.indexOf('error: failed to push') >= 0) {
             const forcePush: vscode.MessageItem = { title: localize('forcePush', 'Force Push') };
             const pushReject: string = localize('localGitPush', 'Push rejected due to Git history diverging.');
             await ext.ui.showWarningMessage(pushReject, forcePush, DialogResponses.cancel);
             // tslint:disable-next-line:no-floating-promises
             localGit.push(remote, 'HEAD:master', { '-f': true });
         } else {
-            throw err;
+            throw parsedError;
```

Comment:
Seems weird to throw parsedError. I would do `throw new Error(parsedError.message);`